### DISCUSSION TITLE
Welcome to ColRadPy Discussions!

### DISCUSSION THREAD

**Discussion by johnson-c** [johnson-c/ColRadPy#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VCMBARC** [comment]
Hi! I want to calculate plasma temperature and density of an argon plasma using the code. I have installed ColRAdPy. Unable to understand how to start now? Which files to be used for argon levels and also how to run colradpy? Please help
